HALO/specmacs/specMACS_polLL_cdsd.yaml

```diff
@@ -0,0 +1,1121 @@
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0202_cb3:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for circle_break 3 (cb3) on 2020-02-02
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200202/EUREC4A_HALO_specMACS-polLL_cdsd_20200202_HALO-0202_cb3_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0202_cl1:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for clover_leg 1 (cl1) on 2020-02-02
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200202/EUREC4A_HALO_specMACS-polLL_cdsd_20200202_HALO-0202_cl1_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0202_ct1:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for clover_turn 1 (ct1) on 2020-02-02
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200202/EUREC4A_HALO_specMACS-polLL_cdsd_20200202_HALO-0202_ct1_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0202_ct2:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for clover_turn 2 (ct2) on 2020-02-02
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200202/EUREC4A_HALO_specMACS-polLL_cdsd_20200202_HALO-0202_ct2_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0202_rct1:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for radar_calibration_tilted 1 (rct1) on 2020-02-02
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200202/EUREC4A_HALO_specMACS-polLL_cdsd_20200202_HALO-0202_rct1_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
+  HALO-0202_rcw1:
+    description: Cloud droplet size distribution derived from polarimetric cloudbow measurements of liquid water clouds for radar_calibration_wiggle 1 (rcw1) on 2020-02-02
+    driver: opendap
+    args:
+      urlpath: https://observations.ipsl.fr/thredds/dodsC/EUREC4A/AIRCRAFT/HALO/SPECMACS/CLOUD_DROPLET_SIZE_DISTRIBUTION/20200202/EUREC4A_HALO_specMACS-polLL_cdsd_20200202_HALO-0202_rcw1_v1.0.nc
+      auth: null
+      chunks: null
+      engine: netcdf4
+
```

Comment:
On some days c1 products are missing, is this just because those flights were too early and the retrieval was not available or are they missing by accident?

Reply:
Yes, some flights were too early and it was too dark to get any measurements. You can also see this in our videos, for example on HALO-0205: http://macsserver.meteo.physik.uni-muenchen.de/campaigns/EUREC4A/flight_HALO-0205.html If the flights were too late in the evening, then c5 or c6 products are also missing.